docs/guides/integration-ctv-guide.md

```diff
@@ -0,0 +1,71 @@
+---
+title: CTV Integration Guide
+sidebar_label: CTV Integration Guide
+pagination_label: CTV Integration Guide
+description: Summary of options for UID2 mobile integration.
+hide_table_of_contents: false
+sidebar_position: 04
+displayed_sidebar: sidebarPublishers
+---
+
+import Link from '@docusaurus/Link';
+
+# CTV Integration Guide
+
+If you're a CTV publisher, there are several ways that you can integrate with UID2 to generate and refresh identity tokens to be passed into the RTB bidstream in the context of your CTV apps.
+
+## Key Integration Steps
+At a high level, to integrate with UID2, you'll implement these three key activities: 
+
+1. Generate the UID2 token.
+1. Refresh the UID2 token as needed.
+1. Pass the UID2 token into the bidstream.
+
+To determine how you'll implement these steps, choose from the [CTV Integration Options](#ctv-integration-options).
+
+## CTV Integration Options
+
+You can decide on the integration option that's best for you based on where you want to generate and refresh the UID2 token, as shown in the following table.
+
+In this table, "client-side" means the CTV application.
+
+| Integration Option | Token Is Generated... | Token Is Refreshed... |
+| :--- | :--- | :--- |
+| [Client-Side Integration](#client-side-integration-for-ctv-apps) | On the client side | On the client side |
+| [Server-Side Integration](#server-side-integration-for-ctv-apps) | On the server side | On the server side |
+| [Client-Server Integration](#client-server-integration-for-ctv-apps) | On the server side | On the client side |
+
+## Client-Side Integration for CTV Apps
+
+The client-side option is for publishers who want to perform both activities on the client side: generate the UID2 token, and then refresh the UID2 token as needed, in the CTV application.
```

Comment:
why _application_? i thought we standardize into _app_?

Reply:
Done all instances